tests/integration/cmd_namespace_test.go

```diff
@@ -183,6 +184,15 @@ ComponentSettings:
 					return out
 				}, 10*time.Second, 1*time.Second).Should(ContainSubstring(commonVar.Project))
 			})
+			It(fmt.Sprintf("should successfully list all the %ss in JSON format", commandName), func() {
+				Eventually(func() bool {
+					ns := helper.Cmd("odo", "list", commandName).ShouldPass().Out()
+					return strings.Contains(ns, commonVar.Project)
+				}).WithTimeout(10 * time.Second).Should(BeTrue())
+
+				out := helper.Cmd("odo", "list", commandName, "-o", "json").ShouldPass().Out()
+				helper.MatchAllInOutput(out, []string{commonVar.Project, "\"active\": true", "\"active\": false"})
```

Comment:
This check is fine but, to me, it does not guarantee that the output is valid JSON. Maybe we could also check that using the `helper.IsJSON` function?

Also, to check that the current active namespace is marked as 'active' in the output, maybe we could rely on some [advanced filtering of `gjson`](https://github.com/tidwall/gjson#path-syntax), by extracting the `active` field of the current project using `namespaces.#[name=="$commonVar.Project"].active` and check that the result is `true`?

Using this would also guarantee that the output is also valid JSON.

Reply:
Makes sense. PTAL 23cc5b2.